airbyte-integrations/connectors/source-woocommerce/source_woocommerce/manifest.yaml

```diff
@@ -1,5 +1,47 @@
 version: "1.0.0"
 
+spec:
+  type: Spec
+  connection_specification:
+    type: object
+    title: Woocommerce Spec
+    $schema: http://json-schema.org/draft-07/schema#
+    required:
+      - api_key
+      - api_secret
+      - shop
+      - start_date
+    properties:
```

Comment:
Let's reorder this in ascending order for consistency. So `api_key` should go first, followed by `api_secret`, `shop`, etc.